### DISCUSSION TITLE
Sound sensor sample code explanation

### DISCUSSION THREAD

**Discussion by RitchieP** [Seeed-Studio/wiki-documents#382] [Q&A] [answered]
I've just started to try and use the sound sensor recently. I come from a non-electronic background.

There's this line of code in the sample sound sensor with Arduino code.

```
for(int i=0; i<32; i++)
```

I want to ask, what is the number 32? Why particularly 32 in this case? 

Thanks for any help in advance!

**Comment by MatthewJeffson** [comment]
Nice to have you onboard! :D 
May I ask which sound sensor you are using now?

**Reply by RitchieP** [reply]
I'm currently using the sensor as in [this link](https://wiki.seeedstudio.com/Grove-Sound_Sensor/)

**Reply by MatthewJeffson** [reply, marked as answer]
Hello! I think this line of code represents the sound sensor reading the sound 32 times in a row and displaying it, for one-time operation. And it doesn't have to be the "32" and you can change it to other number.

**Reply by RitchieP** [reply]
I see now, that clears up the confusion. I think it's also probably best to do some explanation on the code to avoid more confusion in the future.

**Reply by MatthewJeffson** [reply]
Glad to hear that! Thank you for your interests in our product!
